<ui version="4.0" >
 <comment>*********************************************************************
**
** Copyright (C) 1992-$THISYEAR$ Trolltech AS. All rights reserved.
**
** This file is part of the $MODULE$ of the Qt Toolkit.
**
** $LICENSE$
**
** This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
** WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
**
*********************************************************************</comment>
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>FontsPage</class>
 <widget class="QWidget" name="FontsPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>649</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gbDefaultFonts" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Defaul&amp;t Fonts</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Famil&amp;y:</string>
            </property>
            <property name="buddy" >
             <cstring>cmbSubstituteSource</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>&amp;Style:</string>
            </property>
            <property name="buddy" >
             <cstring>cmbFontStyle</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Poin&amp;t Size:</string>
            </property>
            <property name="buddy" >
             <cstring>cmbPointSize</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QComboBox" name="cmbFamily" />
          </item>
          <item>
           <widget class="QComboBox" name="cmbFontStyle" />
          </item>
          <item>
           <widget class="QComboBox" name="cmbPointSize" />
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="leSampleText" >
        <property name="text" >
         <string>Sample Text</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Fonts Su&amp;bstitutions</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>S&amp;ubstitute</string>
          </property>
          <property name="buddy" >
           <cstring>cmbSubstituteSource</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbSubstituteSource" >
          <property name="editable" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>&amp;With</string>
          </property>
          <property name="buddy" >
           <cstring>cmbSubstituteTarget</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbSubstituteTarget" />
        </item>
        <item>
         <widget class="QPushButton" name="pbAdd" >
          <property name="text" >
           <string>A&amp;dd</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="ListWidget" name="lwSubstitutions" />
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>26</width>
              <height>119</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="tbUp" >
            <property name="text" >
             <string>...</string>
            </property>
            <property name="arrowType" >
             <enum>Qt::UpArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbDown" >
            <property name="text" >
             <string>...</string>
            </property>
            <property name="arrowType" >
             <enum>Qt::DownArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>26</width>
              <height>76</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>31</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbRemove" >
          <property name="text" >
           <string>&amp;Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>ListWidget</class>
   <extends>QListWidget</extends>
   <header>listwidget.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbFamily</tabstop>
  <tabstop>cmbFontStyle</tabstop>
  <tabstop>cmbPointSize</tabstop>
  <tabstop>leSampleText</tabstop>
  <tabstop>lwSubstitutions</tabstop>
  <tabstop>tbUp</tabstop>
  <tabstop>tbDown</tabstop>
  <tabstop>pbRemove</tabstop>
  <tabstop>cmbSubstituteSource</tabstop>
  <tabstop>cmbSubstituteTarget</tabstop>
  <tabstop>pbAdd</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
